buddyboss/buddyboss-platform
# Fix #452 Mentions remove issue fixed

### All Submissions:


### How to test the changes in this Pull Request:

1. Go to Activity Page
2. Try "@" mentioning someone
3. Try to delete that in your phone backspace
4. See error

https://drive.google.com/file/d/13o4i4qqoQoLF6zyfCMGgY-O4rGow5VPx/view

### Proof Screenshots or Video



### Other information:




### Changelog entry

> Mentions remove issue fixed
